### ISSUE TITLE
Missing stub for EmptyFieldListFilter admin filter

### ISSUE THREAD

**Issue by haplo** [typeddjango/django-stubs#687]
# Bug report

## What's wrong

`django.contrib.admin.EmptyFieldListFilter` is not included in the stubs. This class was added in Django 3.1. See [https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_filter](ModelAdmin.list_filter) documentation.

To trigger the bug just add a `ModelAdmin` class like this:

```python
from django.contrib import admin

@admin.register(MyModel)
class MyModelAdmin(admin.ModelAdmin):
    list_filter = (
        ("field_name", admin.EmptyFieldListFilter),
    )
```

Then MyPy will complain:

```
django_project/app/admin.py:46: error: Module has no attribute "EmptyFieldListFilter"; maybe "FieldListFilter", "DateFieldListFilter", or "RelatedFieldListFilter"?
```

## How is that should be

No error should be raised.

## System information

- OS: Kubuntu 24.04.2
- `python` version: 3.8.10
- `django` version: 3.2.5
- `mypy` version: 0.812
- `django-stubs` version: 1.8.0
- `django-stubs-ext` version: 0.2.0

**Comment by haplo** [comment]
I will try to submit a PR to fix this.

**Comment by sobolevn** [comment]
Thanks!

**Comment by haplo** [comment]
Oh, turned out you already fixed it in #589 @sobolevn!

Is there a new release planned soon for django-stubs?

**Comment by sobolevn** [comment]
Yes, this month 🙂
